src/app/(reskin)/components/forms/deposit/deposit-form.tsx

```diff
@@ -203,16 +203,24 @@ export const DepositForm = () => {
                 buttonRef={nextButtonRef}
                 onClick={async () => {
                   const result = await validateForm();
```

Comment:
Another difference with the withdrawal one is that here we validate at the top, while there we validate the form after the early returns. If `validateForm` can throw, here we risk failing before reaching the early returns. Should we move `validateForm` right before `const error = result[currentStep];`?

Reply:
it doesn't throw it returns an object with keys being the form field name and values being the error as a string but I can still make that run at the end, aaaand done!